tests/integration/model/modelController.spec.ts

```diff
@@ -369,4 +372,76 @@ describe('ModelController', function () {
       });
     });
   });
+
+  describe('DELETE/models/:identifier', function () {
```

Comment:
Put a space between the words.

Reply:
like this? DELETE / models / : identifier

Reply:
You can see other examples in this file